### PR TITLE
Finish queuing section

### PR DESCRIPTION
**Pull request by daffidwilde**
Add to the general queuing section and write a brief section on queuing and clustering.

### CODE REVIEW

**Comment by daffidwilde** [comment]
@drvinceknight if you get the chance, can you look over the additions in this PR and merge if you're happy?

File: sec/review.tex [hunk 1]
```diff
@@ -438,16 +438,26 @@ \subsection{Healthcare modelling}\label{subsec:healthcare}
 populations~\cite{Foster2019,Gambling2010,Gondek2016,Tsianakas2012} and more
 broadly~\cite{IAPO2012,Richards2015,Santana2019}.
 
+The remainder of this section considers two aspects of healthcare modelling
+research that directly contribute to the advancement of patient-centred care:
```
Comment (drvinceknight) [inline sec/review.tex:442]:
```suggestion
research that directly contributes to the advancement of patient-centred care:
```

File: sec/review.tex [hunk 2]
```diff
@@ -485,65 +495,137 @@ \subsubsection{Segmentation analysis}
...
+The modelling of patient pathways is of great importance to understanding
+patient care in a healthcare system. Recording and studying how a patient moves
+through the system may provide insights into the dynamics of the characteristics
+presented by individuals, and how their true pathway differs from any previously
+defined clinical ideal. Two recent surveys on the literature surrounding the
+modelling of clinical pathways are~\cite{Aspland2019,DeRamonFernandez2019} with
+the latter focusing on process management tools such as process mining.
+
+Often, clustering forms a fundamental part of these methodologies, creating a
+significant overlap. Clustering is a common sense resort in scenarios such as
```
Comment (drvinceknight) [inline sec/review.tex]:
```suggestion
significant overlap. Clustering is a common resort in scenarios such as
```

**Review by drvinceknight — CHANGES_REQUESTED** [review]
Two incredibly minor points. All looks great to me @daffidwilde :+1:

File: sec/review.tex (hunk 1 above)
Comment (daffidwilde) [reply sec/review.tex:442]: I think `contribute` is correct here: "two aspects [...] that *contribute* to..."

**Pull request merged**